### ISSUE TITLE
Missing dependencies - Microsoft.Owin 3.0.1

### ISSUE THREAD

**Issue by mirekholec** [AlexBulankou/ai-stream#3]
AI-Stream (latest 1.0.0) uses NuGet packages:

```
<package id="Microsoft.Owin" version="3.0.1" targetFramework="net45" />
<package id="Microsoft.Owin.Host.SystemWeb" version="3.0.1" targetFramework="net45" />
```

but [NuGet package](https://www.nuget.org/packages/ApplicationInsights.Stream/) requires just:

```
Dependencies
Microsoft.ApplicationInsights (>= 2.0.0)
Microsoft.AspNet.SignalR (>= 2.2.0)
```

After installation AI-Stream I had error

```
Could not load file or assembly 'Microsoft.Owin, Version=3.0.1.0,
```

because I had Microsoft.Owin version 2.1 installed.

I recommend to add missing required dependencies in NuGet package specification.

**Comment by AlexBulankou** [comment]
@mholec Thanks for reporting! Will take a look.

**Comment by AlexBulankou** [comment]
Fixed, released 1.0.1
